release.sh

```diff
@@ -434,43 +432,57 @@ process_module() {
 
     # Run 'make dist/distcheck' to ensure the tarball matches the git module content
     # Important to run make dist/distcheck before looking in Makefile, may need to reconfigure
-    echo "Info: running \"make $MAKE_DIST_CMD\" to create tarballs:"
-    ${MAKE} $MAKEFLAGS $MAKE_DIST_CMD > /dev/null
-    if [ $? -ne 0 ]; then
-	echo "Error: \"$MAKE $MAKEFLAGS $MAKE_DIST_CMD\" failed."
-	cd $top_src
-	return 1
+    if $use_meson; then
```

Comment:
Can we indent the if/else blocks here? It almost caused me to miss the else block entirely...

Reply:
they're indented correctly in the source file but unfortunately we're mixing 4-space indentation with tabs for 8 spaces, and github in its limited wisdom renders tabs at 4 spaces. I'll send a follow-up to fix indentation to spaces only in this file once this is merged to avoid this in the future.